### ISSUE TITLE
Other Dynamic area in Dynamic Pop-Up not read by Screen Reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32515]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

STEPS:
(with screen reader enabled)
1.  Navigate to any dynamic mark using Alt arrow and then press Tab to move to the dynamic pop-up.
2. Arrow down to Other Dynamic button and press enter.
3. Attempt to navigate this new area with arrows, tab, etc.  Note the screen reader goes silent.  I suspect it may still be navigating by keyboard as in other similar issues but not speaking with NVDA.

Similar issues:
#16744 Add Palette 
#16092 Palette More

Some other areas where NVDA stops speaking can be read with OCR, but this one announces "content not visible".

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.5

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
